### DISCUSSION TITLE
Connections/transmissions/netconns when ESP32 is in AP mode

### DISCUSSION THREAD

**Discussion by tommasoclini** [MaJerle/lwesp#133] [Q&A]
How do I start TCP connections, UDP transmissions or netconns when the esp32 is in AP mode?

**Comment by MaJerle** [comment]
I think I explained this already to you. You have to wait client t obe connected first. You get relevant event when this happens. Then you can notify your netconn thread to start.

**Comment by tommasoclini** [comment]
Even with udp transmissions? Sorry if I'm not understanding well(or listening, I will try better this time😅).
The thing is, I would like the esp32 as the ap to start the connection/transmission.
